src/renderer/widgets/WalletDetails/ui/WalletConnectDetails.tsx

```diff
@@ -28,17 +45,27 @@ export const WalletConnectDetails = ({ isOpen, wallet, accounts, onClose }: Prop
   const [isModalOpen, toggleIsModalOpen] = useToggle(isOpen);
 
   const connected = useUnit(walletProviderModel.$connected);
+  const reset = useUnit(walletProviderModel.events.reset);
+  const connect = useUnit(walletConnectModel.events.connect);
+  const reconnectStarted = useUnit(walletProviderModel.events.reconnectStarted);
+  const reconnectAborted = useUnit(walletProviderModel.events.reconnectAborted);
+  const reconnectStep = useUnit(walletProviderModel.$reconnectStep);
 
-  const closeWowModal = () => {
+  useEffect(() => {
+    if (isModalOpen) {
```

Comment:
Looks to me that `reset` should be called on modal close
When model initiates we have a clean state, so there's actually nothing to reset
When we leave we reset so next time when open it's already clean